api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java

```diff
@@ -437,11 +437,115 @@ public CohortDefinition getE2() {
+            + "                              AND e.encounter_type = ${61}     "
+            + "                              AND e.location_id = :location     "
+            + "                              AND (o.concept_id = ${24004} OR o.concept_id = ${2003})     "
+            + "                              AND o.value_coded = ${1065}) AS cont ON p.patient_id = cont.patient_id     "
+            + "      INNER JOIN      "
+            + "      (SELECT p.patient_id, e.encounter_id     "
+            + "                      FROM      "
+            + "                      patient p     "
+            + "                      INNER JOIN      "
+            + "                          encounter e ON e.patient_id = p.patient_id     "
+            + "                      INNER JOIN obs o ON e.encounter_id = o.encounter_id     "
+            + "                      INNER JOIN obs o2 ON e.encounter_id = o2.encounter_id     "
+            + "                      WHERE     "
+            + "                          p.patient_id = e.patient_id     "
+            + "                          AND p.voided = 0     "
+            + "                              AND e.voided = 0     "
+            + "                              AND o.voided = 0     "
+            + "                              AND e.encounter_type = ${61}     "
+            + "                              AND e.location_id = :location     "
+            + "                              AND o.concept_id = ${24005}     "
+            + "                              AND o.value_coded = ${1065}     "
+            + "                              AND o2.concept_id = ${24006}     "
+            + "                              AND o2.obs_datetime between :startDate AND :endDate      "
+            + "                              UNION     "
+            + "                              SELECT p.patient_id, e.encounter_id     "
+            + "                      FROM      "
+            + "                      patient p     "
+            + "                      INNER JOIN      "
+            + "                          encounter e ON e.patient_id = p.patient_id     "
+            + "                      INNER JOIN obs o ON e.encounter_id = o.encounter_id     "
+            + "                      INNER JOIN obs o2 ON e.encounter_id = o2.encounter_id     "
+            + "                      WHERE     "
+            + "                          p.patient_id = e.patient_id     "
+            + "                          AND p.voided = 0     "
+            + "                              AND e.voided = 0     "
+            + "                              AND o.voided = 0     "
+            + "                              AND e.encounter_type = ${61}     "
+            + "                              AND e.location_id = :location     "
+            + "                              AND o.concept_id = ${24011}     "
```

Comment:
I cant see this in the specs

Reply:
https://docs.google.com/document/d/159ZGr5KJcw8tkb-FXKncBxVgplYJFMOU/edit This is the spec